ui/app/AppLayouts/Wallet/stores/WalletAssetsStore.qml

```diff
@@ -18,6 +18,25 @@ QtObject {
     readonly property var assetsController: ManageTokensController {
         sourceModel: groupedAccountAssetsModel
         settingsKey: "WalletAssets"
+        serializeAsCollectibles: false
+
+        // TODO #13313: call the assets controller for all events
+        onRequestSaveSettings: (jsonData) => {
+            // savingStarted()
```

Comment:
why in this case this started/finished pair is not used?

Reply:
They are already done serialy in the internal `saveToQSettings` as was done before. The marker methods started/finished are just for external storage as discussed above.